Fix Docker build for frontend

Currently the FE CI is failing with the following error:

```
 invalid argument "/stamacasa:1c4d3971fff8332e95d214edb0826c7ad03c245a" for "-t, --tag" flag: invalid reference format
See 'docker build --help'.
##[error]Process completed with exit code 125.
```

--- a/.github/workflows/dockerimage.api.yml
+++ b/.github/workflows/dockerimage.api.yml
@@ -17,7 +17,7 @@ on:
 name: CI-API
 
 env:
-  IMAGE_NAME: ${{ secrets.DOCKER_HUB_ORGANIZATION }}/stamacasa-api
+  IMAGE_NAME: ${{ secrets.DOCKER_HUB_ORGANIZATION || 'code4romania' }}/stamacasa-api
 
 jobs:
   build-parser:

--- a/.github/workflows/dockerimage.frontend.yml
+++ b/.github/workflows/dockerimage.frontend.yml
@@ -15,7 +15,7 @@ on:
 name: CI-FrontEnd
 
 env:
-    IMAGE_NAME: ${{ secrets.DOCKER_HUB_ORGANIZATION }}/stamacasa
+    IMAGE_NAME: ${{ secrets.DOCKER_HUB_ORGANIZATION || 'code4romania' }}/stamacasa
 
 jobs:
     build-parser:

--- a/.github/workflows/dockerimage.identityserver.yml
+++ b/.github/workflows/dockerimage.identityserver.yml
@@ -17,7 +17,7 @@ on:
 name: CI-IdentityServer
 
 env:
-    IMAGE_NAME: ${{ secrets.DOCKER_HUB_ORGANIZATION }}/stamacasa-identityserver
+    IMAGE_NAME: ${{ secrets.DOCKER_HUB_ORGANIZATION || 'code4romania' }}/stamacasa-identityserver
 
 jobs:
     build-parser:

